lua/telescope/builtin/lsp.lua

```diff
@@ -246,6 +276,7 @@ local feature_map = {
   ["code_actions"]      = "code_action",
   ["document_symbols"]  = "document_symbol",
   ["references"]        = "find_references",
+  ["definitions"]        = "goto_definition",
```

Comment:
```suggestion
  ["definitions"]       = "goto_definition",
```
Aesthetic purposes XD